packages/mgt-components/src/components/mgt-file-list/mgt-file-upload/strings.ts

```diff
@@ -21,9 +21,9 @@ export const strings = {
   buttonOk: 'OK',
   buttonCancel: 'Cancel',
   buttonUpload: 'Upload',
-  buttonKeep: 'Keep',
+  buttonKeep: 'Keep Both',
```

Comment:
_nitpick:_ we should prefer sentence casing
```suggestion
  buttonKeep: 'Keep both',
```